config/examples/settings.yml

```diff
@@ -11,7 +11,6 @@ base: &default
   asset_host: <%= ENV.fetch('RAILS_ASSET_HOST', nil) %>
   daily_weekly_reports_pref: <%= ENV.fetch('ENABLE_DAILY_WEEKLY_REPORTS_PREF', '1') == '1' %>
   readonly_custom_domains_settings: <%= ENV.fetch('READONLY_CUSTOM_DOMAINS_SETTINGS', '1') == '1' %>
-  airbrake_api_key: <%= ENV['AIRBRAKE_API_KEY'] %>
```

Comment:
I think u will have to remove it also from the deploy repo :smile:

Reply:
https://github.com/3scale/deploy/pull/621